README.md

```diff
@@ -1,6 +1,80 @@
-ASP.NET Core IIS Integration
-========
-
-This repo hosts the ASP.NET Core middleware for IIS integration.
+This repo hosts the ASP.NET Core middleware for IIS integration and the AspNetCoreModule
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
```

Comment:
@shirhatti I think I should just remove this section of the README and we can update it later.